### DISCUSSION TITLE
Wild card SP metadata file..

### DISCUSSION THREAD

**Discussion by kuwars98** [latchset/mod_auth_mellon#80] [Q&A] [answered]
Hello, 

I'm planning to implement mod_auth_mellon for SAML base authentication on apache reverse proxy but we have 100+ URLs to migrate on SAML SSO authentication but instated of multiple SP metadata file, Shall we use single metadata(means wild card SP metadata) file ?
Is it possible in Mellon? Please help me on this

**Comment by kuwars98** [comment]
Hello,

anyone can help me ?

**Comment by simo5** [comment, marked as answer]
I do not think this is possible in SAML in general.
The best way to deal with something like that is to create an authentication domain instead, and redirect clients to a common authentication domain, with a return path that will bring the client back to the correct site after authorization. The authorization can then be carried on as a cross site cookie or other mechanism within the receiving page.

**Reply by kuwars98** [reply]
Thank you for update
